Firestore/Source/Local/FSTLevelDBMigrations.mm

```diff
@@ -102,22 +145,37 @@ + (FSTLevelDBSchemaVersion)schemaVersionWithTransaction:
 }
 
 + (void)runMigrationsWithTransaction:(firebase::firestore::local::LevelDbTransaction *)transaction {
+  [self runMigrationsWithTransaction:transaction upToVersion:kSchemaVersion];
+}
+
++ (void)runMigrationsWithTransaction:(firebase::firestore::local::LevelDbTransaction *)transaction
```

Comment:
This method should perhaps take an `FSTLevelDB` rather than a `LevelDbTransaction`. Since we are not running everything within a single transaction, I don't think it makes sense for that to be what is passed in.

Reply:
Done.